### ISSUE TITLE
-- lower priority -- Parallelization over number of molecules instead of trials per molecule

### ISSUE THREAD

**Issue by marcelmbn** [grimme-lab/MindlessGen#56]
Currently, the parallelization exploits parallel approaches for generating a molecule. However, the bottleneck is the finalization of the last calculations once a compound is found.
This would be overcome by searching in parallel for different molecules of `num_molecules`.

## Alternative

Break the ongoing calculations (`xtb` or `orca`) once a compound is found.
